packages/unmock-core/src/__tests__/validator.test.ts

```diff
@@ -0,0 +1,179 @@
+import { Response, Schema } from "../service/interfaces";
+import { StateRequestValidator } from "../service/validator";
+
+const schema: Schema = {
+  properties: {
+    test: {
+      type: "object",
+      properties: {
+        id: {
+          type: "integer",
+          format: "int64",
+        },
+      },
+    },
+    name: {
+      type: "string",
+    },
+    foo: {
+      type: "object",
+      properties: {
+        bar: {
+          type: "object",
+          properties: {
+            id: {
+              type: "integer",
+            },
+          },
+        },
+      },
+    },
+    tag: { type: "string" },
+  },
+};
+const response: Response = {
+  content: {
+    "application/json": {
+      schema,
+    },
+  },
+  description: "foo bar",
+};
+
+describe("Tests findStatePathInService", () => {
+  it("with empty path", () => {
```

Comment:
I think it would be quite helpful for future test readers if the name of the test told what the function is expected to do, something like `describe("finding state path in service")`,  `it("should produce empty state from empty path")`.